java/org/apache/catalina/connector/Response.java

```diff
@@ -1069,16 +1069,20 @@ public void sendError(int status, String message) throws IOException {
             return;
         }
 
-        setError();
+        if(103 == status) {
```

Comment:
Are we not putting a space before the parenthesis?

Reply:
Oops I'll fix that. Forgot to run validate.